doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -97,8 +97,12 @@ int main(int argc, char** argv)
     ROS_FATAL_STREAM("Could not find planner plugin name");
   try
   {
-    planner_plugin_loader.reset(new pluginlib::ClassLoader<planning_interface::PlannerManager>(
-        "moveit_core", "planning_interface::PlannerManager"));
+    planner_plugin_loader.reset(new pluginlib::ClassLoader<planning_interface::PlannerManager>("moveit_core", "planning"
+                                                                                                              "_interfa"
+                                                                                                              "ce::"
+                                                                                                              "PlannerM"
+                                                                                                              "anage"
+                                                                                                              "r"));
```

Comment:
@henningkayser @tylerjw @rhaschke this clang format result looks horrible for our public-facing tutorials. Aside from the clang-off tags, is there some tweak we can make to have it not do this??

Reply:
@JafarAbdi This PR uses the wrong clang-format config

Reply:
See https://github.com/ros-planning/moveit_tutorials/pull/519